### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by prositen**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/prositen/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
+
+**Thanks for letting us show you around!**
+
+What you’re seeing here is a GitHub pull request with the code diffs
```
Comment (prositen) [inline ReadMe.md]: !resolve

Comment (prositen) [inline ReadMe.md]: derp

Comment (prositen) [inline ReadMe.md]: !resolved

**Pull request merged**